docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment
+sidebar_label: Controller
+id: deploy
+---
+
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md';
+
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
+
+## Installation
+
+The controller package `openziti-controller` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a PKI, config, and database by installing interactively with prompts or by setting these values.
+
+1. In `/opt/openziti/etc/controller/service.env` (the answer file for the service)

```

Comment:
I'd like to remove any 'opt-in' stuff and tell them exactly what to set and how to execute it from a minimal perspective. We know there are really just one thing that one really MUST get right from the start, the advertised address/port for the controller (which must support IP-only deployments). For a new install we should just tell them this is vitally important because they are about to generate a PKI (by running some command/commands we provide for them to run).

Reply:
Raised an issue for supporting IPv4 addresses: https://github.com/openziti/ziti/issues/1993 and documenting that option in https://github.com/openziti/ziti-doc/issues/833.

Reply:
If I understand your preference to remove information about opt-in to bootstrapping, then I'm unsure how to offer both alternatives. This section is about providing or generating a configuration (your preferred umbrella term for bootstrapping). If they provide a configuration, then the default service configuration works (no opt-in to bootstrapping). If they are not providing any one of the three pillars of the configuration, then they must opt-in to bootstrapping by setting the var in service.env.